### ISSUE TITLE
Add more sites to the scrapy start_urls

### ISSUE THREAD

**Issue by groovecoder** [codefortulsa/city-struggle-bus#6]
With just nextcity.org and govtech.com, I was able to crawl and index 18.8k articles/pages. @chimchim237 knows the bigger list of real sites to crawl.

**Comment by chimchim237** [comment]
@groovecoder i made a REALLY good list.  ...and I thought I emailed it to you?  but now i can't find it. :(

**Comment by groovecoder** [comment]
Yeah, I thought I saw that somewhere too ... but I can't find it in email. :cry: Re-creating on GH issue will keep it public and permanent at least.

**Comment by chimchim237** [comment]
nextcity.org
strongtowns.org
citylab.com
iqc.ou.edu
urbanland.uli.org
planetizen.com
streetsblog.net
governing.com

cnt.org/blog
thehappycity.com/blog/
smartgrowthamerica.org/blog
smartgrowthtulsa.com/blog
100resilientcities.org/blog
sunlightfoundation.com/blog
brookings.edu/about/programs/metro/research

**Comment by chimchim237** [comment]
I found the file to edit, and can easily do that if you want me to, this weekend.  Lemme know.

**Comment by chimchim237** [comment]
found another great one.
http://na.smartcitiescouncil.com/
http://cc.smartcitiescouncil.com/

**Comment by chimchim237** [comment]
Another one:
http://datasmart.ash.harvard.edu/
